w03/d2/morning_exercise/index.html

```diff
@@ -5,9 +5,17 @@
     <title></title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.0/animate.min.css">
     <link rel="stylesheet" href="main.css">
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.0/animate.min.css">
   </head>
   <body>
+<<<<<<< HEAD
+    <div class = "type stuff">
+      <img  width="500px" src="https://i.kym-cdn.com/photos/images/original/001/233/039/e89.gif" alt="hacking">
+
+    </div>
+=======
     <img class="animated heartBeat faster infinite" width="500px" src="https://i.kym-cdn.com/photos/images/original/001/233/039/e89.gif" alt="hacking">
+>>>>>>> 1219ca81c1c9d511ddaee3d19764dd1e447d3c23
```

Comment:
Always make sure to clean these merge markers up when you fix merge conflicts. This can cause issues for teammates when they try to work with your code.